src/pages/questionPage.js

```diff
@@ -105,5 +104,5 @@ const nextQuestion = () => {
   setTimeout(() => {
     initQuestionPage();
     currentAnswerElement.classList.remove(addClass);
-  }, 1500);
+  }, 300);
```

Comment:
300 is maybe a bit too low? Hard to process what the right answer was in such a short time. Also would be good to move this value to a constant.